draft-ietf-quic-qpack.md

```diff
@@ -682,10 +682,10 @@ acknowledged as this instruction does not insert an entry.
 An encoder adds an entry to the dynamic table where the header field name
 matches the header field name of an entry stored in the static or the dynamic
 table using an instruction that starts with the '1' one-bit pattern.  The second
-(`S`) bit indicates whether the reference is to the static or dynamic table. The
+('T') bit indicates whether the reference is to the static or dynamic table. The
```

Comment:
T seems odd to me because they both use a table. And then we have T=1 meaning that it's the static table.

Are we really so constrained that we can't assign a noun to bits?

Reply:
My main gripe was that we were using S with two different meanings (Sign bit and "Static Table bit"). On slack, @afrind was ok with T, so I went with that. I see it as a table "index" personally, which makes sense to me. 

But really, I don't care which letter or noun is used, as long as the same letter/noun isn't used for different things.

Reply:
I'm fine with it being an index of the tables.  In my mind (thanks in part to HPACK) the static table comes "first," so it feels a little weird that it's 1 vs. 0, but that's probably better than turning something purely editorial into a needless breaking change at this point.

Reply:
I'm fine with T.  T=1 -> Static is a little weird I guess, but you can just think about it as the sTatic table bit.